slurmpilot/config.py

```diff
@@ -88,7 +85,7 @@ def remote_slurmpilot_path(self, cluster: str | None = None) -> Path:
         if cluster in self.cluster_configs:
             return Path(self.cluster_configs[cluster].remote_path)
         else:
-            return Path(self.general_config.remote_path)
+            return Path(ClusterConfig.remote_path)
```

Comment:
Should be the following right? If so it means that this branch is not tested anymore (not sure if it was before).
```suggestion
            return Path(ClusterConfig().remote_path)
```

Reply:
I thought so too, but accessing the value of the class attributes without instantiating the class works :)